textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/Span.java

```diff
@@ -3,22 +3,40 @@
 public class Span {
     //The name of the field (in the tuple) where this span is present
     private String fieldName;
-    //The start of the span
+    //The start of the span. It is the position of the first character of span in the document.
     private int start;
-    //The end of the span
+    //The end of the span.It is the position of the first character of span in the document

```

Comment:
Change it to "The end position of the span, which is the offset of the gap after the last character of the span."